WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailToolbar.swift

```diff
@@ -34,9 +34,12 @@ class ReaderDetailToolbar: UIView, NibLoadable {
 
     weak var delegate: ReaderDetailToolbarDelegate? = nil
 
+    private lazy var readerImprovementsEnabled: Bool = {
+        return RemoteFeatureFlag.readerImprovements.enabled()
+    }()
```

Comment:
Great idea DRYing the access to the feature flag 😄 

 Just curious: Did you consider injecting it at init time? I can see how this being a `UIView` subclass could make it tricky.